spec/lib/ordering_helper_spec.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+require 'spec_helper'
+
+def stub_ordering_helper(helper, eq_model1, eq_model3)
+  allow(helper).to receive(:ordering).and_return(3)
+  allow(helper).to receive(:predecessor).and_return(eq_model1)
+  allow(helper).to receive(:successor).and_return(eq_model3)
+  allow(helper).to receive(:category_orderings).and_return([1, 3, 5])
+  allow(helper).to receive(:successors).and_return([eq_model3])
+end
+
+def stubbed_eq_model(ord = nil)
+  eq_model = FactoryGirl.build_stubbed(:equipment_model, ordering: ord || nil)
+  allow(eq_model).to receive(:update_attribute)
+  eq_model
+end
+
+describe OrderingHelper do
+  describe 'category information' do
```

Comment:
these are tests on private methods whose results we don't rely on outside of the class -- they can be removed